src/internal/globalVars.ts

```diff
@@ -24,6 +24,6 @@ export class GlobalVars {
   public static backButtonPressHandler: () => boolean;
   public static beforeUnloadHandler: (readyToUnload: () => void) => boolean;
   public static changeSettingsHandler: () => void;
-  public static onStartConversationHandler: (sunEntityId: string, conversationId: string) => void;
-  public static onCloseConversationHandler: (sunEntityId: string, conversationId?: string) => void;
+  public static onStartConversationHandler: (sunEntityId: string, conversationId: string, threadId: string, entityId: string) => void;
```

Comment:
make sure you are returning these fields in same order in teams code

Reply:
subEntityId, right? (typo)

Reply:
subEntityId was already there @aosolis . Adding entity Id now, because from the personal scope we will need to know which tab the user wants to start a conversation in